validation-test/compiler_crashers_fixed/28625-swift-typechecker-lookupmember.swift

```diff
@@ -0,0 +1,17 @@
+// This source file is part of the Swift.org open source project
+// Copyright (c) 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+// RUN: not %target-swift-frontend %s -typecheck
+protocol Element
```

Comment:
Instead of editing the existing crashers can you add a new test or update an existing test with expected-error {{ }} etc? I can't tell if this is supposed to type check or not.